File: 344-ReverseString/344-ReverseString.cpp
// Last updated: 3/30/2026, 2:13:20 PM
class Solution {
public:
    void reverseString(vector<char>& s) {
        int left = 0, right = s.size() - 1;
        while(left<right){
            char temp = s[left];
            s[left] = s[right];
            s[right] = temp;
            left++;
            right--;
        }
    }
};

File: 344-ReverseString/344-ReverseString.java
// Last updated: 3/30/2026, 2:13:14 PM
class Solution {
    public void reverseString(char[] arr) {
        int left = 0, right = arr.length - 1;
        while(left < right){
            char temp = arr[left];
            arr[left] = arr[right];
            arr[right] = temp;
            left++;
            right--;
        }
    }
}